### ISSUE TITLE
V Rising (1604030)

### ISSUE THREAD

**Issue by kdwinter** [ValveSoftware/Proton#5845]
# Compatibility Report
- Name of the game with compatibility issues: V Rising
- Steam AppID of the game: 1604030

## System Information
- GPU: RX 6700 XT
- Driver/LLVM version: Mesa 22.0.3
- Kernel version: 5.17.1-zen1-1-zen
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/kdwinter/7b6fbfe8a3a78ffe8ceb93214d2cfa23
- Proton version: Experimental

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 

Crashes on startup. With `PROTON_USE_WINED3D=1` the game *sometimes* reaches the main menu, but then again crashes after the intro video (which isn't visible; only artifacts ), or upon attempting to join a server. With DXVK, it consistently crashes before reaching any menus.

Different Proton versions do not appear to change this behavior, except for 6.3: with Proton 6.3 the game consistently starts, but then stops on a "Steamworks initialization failed" screen.

## Reproduction

Launch the game from Steam.

[steam-1604030-2.log](https://github.com/ValveSoftware/Proton/files/8709118/steam-1604030-2.log)

[steam-1604030.log](https://github.com/ValveSoftware/Proton/files/8709114/steam-1604030.log)

**Comment by kdwinter** [comment]
Steam log:
[steam-1604030-2.log](https://github.com/ValveSoftware/Proton/files/8709129/steam-1604030-2.log)

**Comment by screwylightbulb** [comment]
Can confirm that with recent GE builds the game similarly to the OP starts and then fails with "Steamworks initialization failed".

**Comment by calvinbull** [comment]
I can also confirm this behaviour with all recent proton and GE builds. Installs fine but crashes to desktop immediately on launch.

OS: Pop!_OS 22.04 LTS
KERNEL: 5.17.8-xanmod1-x64v2
CPU: AMD Ryzen 7 3800X 8-Core
GPU: NVIDIA GeForce RTX 2080 Ti
GPU DRIVER: NVIDIA 510.68.02
RAM: 32 GB

**Comment by deverebor** [comment]
Same issue here.

**OS:** Pop!_OS 22.04 
**LTSKERNEL:** 5.17.5-76051705-generic
**CPU:** Intel Core i5-10300H @ 2.50GHz
**GPU:** NVIDIA GeForce GTX 1650
**GPU DRIVER:** NVIDIA 510.68.02
**RAM:** 16 GB

Using Proton-7.0rc2-GE-1 starts and then fails with "Steamworks initialization failed".

![image](https://user-images.githubusercontent.com/62480586/168856963-764201af-75ce-4fd7-adc4-3884198be726.png)

**Comment by hatani** [comment]
After quick post on Reddit, few more people confirming same issue (crashing silently before or in loading screen).  I am having same issue as well.

Fedora 36
AMD Ryzen 5
AMD 6800 XT
OpenGL version string: 4.6 (Compatibility Profile) Mesa 23.0.0-git
RAM 32 GB

**Comment by jimsjoh** [comment]
Confirmed, same issue

**Comment by Dreameh** [comment]
Confirmed, same issues.

**OS:** Arch
**Kernel:** 5.17.7-arch1-2
**CPU:** AMD Ryzen 5 3600XT
**GPU:** AMD 5700 XT
**RAM:**  32 GB

**Comment by screwylightbulb** [comment]
Have at least one report of the "Steamworks initialization failed" thing happening on Windows too. 

https://steamcommunity.com/app/1604030/discussions/0/5154944131336583253/

**Comment by static-glitch** [comment]
Same crash on 7.0-2, Experimental, and GE 7-17. On Proton 6.3-8, I get the SteamWorks initialization problem, but I see others on the Discord having the SteamWorks problem on Windows as well.

CPU: AMD Ryzen 9 5950X
RAM: 128 GB 
Ubuntu 22.04 LTS (64 bit)
Kernel Version:  5.15.0-27-generic
X Server Release:  12101003
X Window Manager:  KWin
Steam Runtime Version:  steam-runtime_0.20220315.0
Video Card: NVIDIA GeForce GTX 1080
Driver Version:  4.6.0 NVIDIA 510.60.02

FWIW, it was also doing this on the closed beta, but I was only in the last phase. I see Discord posts that say it worked in earlier stages of the closed beta, but don't have personal experience with it. I don't know if it was something the game devs updated, or a minor update to Proton that happened to break it (e.g. maybe 7.0-1 to 7.0-2?)

**Comment by screwylightbulb** [comment]
Proton Experimental now seems to be able to get past the "Steamworks initialization failed" thing, but now once you start a server, it gets stuck on a screen that says "Connecting" and then crashes after a while. Apologies I am not able to post a log right now.

**Comment by 1player** [comment]
Still not starting for me with Proton Experimental [bleeding-edge]

**Comment by hatani** [comment]
> Still not starting for me with Proton Experimental [bleeding-edge]

Same, but after updating bleeding-edge Proton Experimental just now, I can at least stop hung game without it hanging whole Steam.

Also it seems Proton 6.21-GE-2 is the only one letting me see the loading screen with Steamworks initialization error (same as screenshot above from other user).

**Comment by dbyzero** [comment]
Same here

**Comment by xuerivera** [comment]
Can't luanch the game at all so far I've got a blue screen and then crashes again

Proton 7.0-1
Distro:Pop!_OS 22.04 LTS
Kernel:5.17.5-76051705-generic
RAM:16 GB
GPU Driver:NVIDIA 510.68.02
GPU:NVIDIA GeForce GTX 1070
CPU:Intel Core i5-10600K @ 4.10GHz

**Comment by screwylightbulb** [comment]
This is a log of the game for when the game actually gets to the screen that says Steamworks initialization failed. 

[steam-1604030.zip](https://github.com/ValveSoftware/Proton/files/8730566/steam-1604030.zip)

**Comment by nenoro** [comment]
Can confirm it launches, crashes when joining a server then stop launching

Proton 7.0-1, GE-7, Experimental
Distro: Gentoo
Mesa-git
Kernel: 5.17.7-xanmod1-pineapple
RAM: 28 GB
GPU Driver: xf86-video-amdgpu
GPU: RX 5700
CPU: R7 1800X, 3.6 Ghz

**Comment by DeeV55** [comment]
Managed to get into the menu one time but it crashed trying to get into the game:
https://imgur.com/a/57ekCBx

Proton experimental [Bleeding edge]
Distro: Arch Linux
Kernel: 5.17.7-arch1-1
RAM: 16 GB
GPU Driver: NVIDIA 515.43.04
GPU: NVIDIA GeForce RTX 2070 SUPER
CPU: AMD Ryzen 5 3600 6-Core

**Comment by screwylightbulb** [comment]
This log is Proton Experimental bleeding-edge, opens on white screen and then crash. RX 6700 XT using radv.

[VRising.zip](https://github.com/ValveSoftware/Proton/files/8743372/VRising.zip)

**Comment by LordJakki** [comment]
It seems someone got it working by using Windows version of Steam and Bottles
https://www.reddit.com/r/linux_gaming/comments/uuary2/v_rising_arch_working_minor_gfx_issues/?utm_source=share&utm_medium=web2x&context=3

**Comment by 1player** [comment]
Confirmed. Running Steam through Wine is super buggy and not a fun experience, but I was able to start the game and play, so it's definitely a Proton issue.

**Comment by screwylightbulb** [comment]
Tried using Bottles+Windows Steam and the latest caffe runner. Seems to work more or less, but on the screen that says "Press space to wake up" it seems to not be taking and input. Particle effects are still going so it's not a crash.

**Comment by Melkren** [comment]
Worked for me using bottles. Also tested on Steam Deck. Had no problems pressing space to wake up either with keyboard and mouse plugged in.

**Comment by WelshPixie** [comment]
> Worked for me using bottles. Also tested on Steam Deck. Had no problems pressing space to wake up either with keyboard and mouse plugged in.

Which runner did you use?

**Comment by Melkren** [comment]
> > Worked for me using bottles. Also tested on Steam Deck. Had no problems pressing space to wake up either with keyboard and mouse plugged in.
> 
> Which runner did you use?

I just followed the guide posted above by LordJakki including the edit at the bottom of the post

**Comment by WelshPixie** [comment]
> > > Worked for me using bottles. Also tested on Steam Deck. Had no problems pressing space to wake up either with keyboard and mouse plugged in.
> > 
> > 
> > Which runner did you use?
> 
> I just followed the guide posted above by LordJakki including the edit at the bottom of the post

Weird, same - I'm the one that can't press space to wake up (screwylightbulb was posting on my behalf)

**Comment by LordJakki** [comment]
For me it just gets stuck on loading screen... if I move mouse i can hear sound effects of buttons and by clicking I can open Terms of Service and Privacy policy but screen just stays on loading image.

**Comment by Wolf-086** [comment]
I tried the windows version of steam yesterday, installed it via Lutris. The recommended settings for my game were high, but no matter the settings I used, the game ran like shit. I spent 4-5 hours yesterday working on it, but deemed it wasn't worth my time because the poor performance.

**Comment by Wolf-086** [comment]
Just tried with bottles. Steam won't move past steam guard. Followed everything to a T. As a Unity dev myself, I dont understand why they just wont port it to Linux.

**Comment by nenoro** [comment]
yeah steam guard doesn't pop up for me too

**Comment by 1player** [comment]
> Just tried with bottles. Steam won't move past steam guard. Followed everything to a T. As a Unity dev myself, I dont understand why they just wont port it to Linux.

I disabled Steam Guard to make it work. You can also try enabling the Virtual Desktop feature in Bottles, it seems to render the steam guard screen correctly. You'll want to turn it off again when actually playing the game.

**Comment by Wolf-086** [comment]
> 

It's not that I have an issue getting steam guard to come up. Once I enter in my code, steam crashes. I tried running it as a virtual desktop, I tried disabling steam guard, the result is the same.

**Comment by kisak-valve** [comment]
While it's okay to discuss most workarounds here, please keep in mind that this issue tracker is intended for Proton feedback. Issues with Steam for Windows run with wine/bottles is out of scope for this issue tracker and should be discussed elsewhere.

**Comment by Melkren** [comment]
I think this is relevant.

I used the bottles workaround to get it working and it ran fine through desktop mode on the Steam Deck that way. 

I then managed to add the game to Steam as a none steam game using this guide:
https://github.com/oh-nyoo/steam-deck-advance/wiki/Installing-Non-Steam-Windows-Games-to-Steam

Then when I ran it in game mode, the intro was a white screen again and I got the "Steamworks Initialization Failed" error.

I have no idea how steam handles running things when you add them this way, but it seems very strange that this same issue occurs again after running it this way.

**Comment by screwylightbulb** [comment]
It does narrow down the problem to Steamworks via Proton specifically though.

**Comment by abique** [comment]
I don't think that this thread needs more info today.
No one can use V-Rising on Linux, which means that they can reproduce this issue very easily, and work until they get the game playable.

**Comment by Nicknakin** [comment]
I mean they showed an example of someone getting it working in bottles.

**Comment by WelshPixie** [comment]
https://steamcommunity.com/app/1604030/discussions/0/3272440173689684567/?tscn=1653172869#c3272440173692073685

Confirmed that copying stun_steamnetworking.dll from Battlerite Royale into the two specified V Rising folders works - game now launches and plays with Proton 7.0-2

**Comment by iMonZ** [comment]
> https://steamcommunity.com/app/1604030/discussions/0/3272440173689684567/?tscn=1653172869#c3272440173692073685
> 
> Confirmed that copying stun_steamnetworking.dll from Battlerite Royale into the two specified V Rising folders works - game now launches and plays with Proton 7.0-2
Could you share the file please?

**Comment by 1player** [comment]
I don't think this is a good place to share binaries from strangers. You need to download *Battlerite Royale* from Steam to get that file, and it's free to play.

**Comment by fonikz** [comment]
I have copied the DLL files, but the game is still crashing while attempting to start a solo session.

**Comment by GloriousEggroll** [comment]
Copy DLL method works here. Private PvE session works also.

Steps:

1. Download free to play Battlerite Royale

2. Copy from: 
Battlerite Royale/BattleriteRoyale_Data/Plugins/stun_steamnetworking.dll

To both of these locations:

VRising/VRising_Data/Plugins/x86_64/
VRising/VRising_Server/VRisingServer_Data/Plugins/x86_64/

3. To fix intro video, launch game with:
`WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%`

![vrising](https://user-images.githubusercontent.com/11287837/169751088-fe502050-2a74-4bf5-bd54-e6c1adae4e30.png)

**Comment by fonikz** [comment]
The last 103 lines of my Proton log are the same except for the timestamp:

`467.414:0130:0134:fixme:system:QueryDisplayConfig (00000002 000000000010F740 00000000E0000010 000000000010F8C8 00000000E00000B0 0000000000000000): semi-stub`

It's crashing either during Server Starting Up XXX/209, or during Loading.

**Comment by hrkrx** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5845#issuecomment-1134205419

~~sadly this doesn't seem to work on the steamdeck~~ nvm I did not use the Royale version

**Comment by GStefanowich** [comment]
> ~sadly this doesn't seem to work on the steamdeck~ nvm I did not use the Royale version

Ninja edit'ed before I could say anything.

Yeah to anyone else make sure you're using [Battlerite (Royale)](https://store.steampowered.com/app/879160/Battlerite_Royale/) and *not* [Battlerite](https://store.steampowered.com/app/504370/Battlerite/).

Saw this kind of clarification needed on Steam Discussions and in Reddit comments.

**Comment by gofman** [comment]
Should hopefully be fixed now in Proton Experimental [bleeding edge] branch (without requiring any manual workarounds).

**Comment by luthes** [comment]
> Should hopefully be fixed now in Proton Experimental [bleeding edge] branch (without requiring any manual workarounds).

What branch are you building from? I don't see any recent changes committed

**Comment by gofman** [comment]
> What branch are you building from? I don't see any recent changes committed

https://github.com/ValveSoftware/Proton/commits/experimental-bleeding-edge-7.0-16382-20220523-pad54a5-w198163-d51e56c-v910f15

You can select Proton Experimental properties, choose Beta property page and select [bleeding-edge] branch there, no need to build it.

**Comment by hatani** [comment]
I confirm current Proton Experimental bleeding-edge let's you get into game (online play also).  However, there are two issues:
1) problem with graphics for me 
![image](https://user-images.githubusercontent.com/740982/169973833-6de7a19c-530f-49ef-b746-4dc947a24951.png)

2) you still need to kill the game on exit (it's not responding).

BONUS 3) Another interesting fact (not related to our issue) - you don't see any of your existing characters (as played via Bottles workaround) BUT - if you connect to same server, you will be loaded into old character :).

**Comment by GStefanowich** [comment]
> Another interesting fact (not related to our issue) - you don't see any of your existing characters (as played via Bottles workaround)

The game doesn't utilize steam cloud saving for your private saves, so running the game through Bottles and regular Proton don't sync up, they're saved in different locations.

Use something like `mlocate` to find and copy your saves from Bottles.

> $: `updatedb`
> $: `locate -i rising | grep -i save`

**Comment by screwylightbulb** [comment]
> I confirm current Proton Experimental bleeding-edge let's you get into game (online play also). However, there are two issues:
> 
>     1. problem with graphics for me

Need more info here. GPU etc.

**Comment by hatani** [comment]
> 

Sorry, I posted my specs above, but here they are again, if anything is missing, let me know, I'll try to fill-in.

Fedora 36
AMD Ryzen 5
AMD 6800 XT
OpenGL version string: 4.6 (Compatibility Profile) Mesa 23.0.0-git
RAM 32 GB
Using Wayland

I repeat, via Bottles (wine Steam + wine apparently), game runs PERFECTLY for me (only problem being needing to kill when exiting to OS).

**Comment by Seibz** [comment]
Thanks for adding those to Experimental bleeding edge. Quite the enjoyable game. Looks like all that's left to fix is needing to stop the game from Steam on exit.

**Comment by gofman** [comment]
Hang on exit should hopefully be fixed too now in the latest Proton Experimental [bleeding-edge].

**Comment by abique** [comment]
You are doing an amazing job :)

**Comment by fishd72** [comment]
Can confirm that using the latest Proton Experimental [bleeding-edge] the game launches, is playable and exits cleanly.

I'm having an issue though where if the graphics options in-game are set to "Exclusive Fullscreen", then alt-tabbing out of the game to the desktop, and then attempting to return to the game using alt-tab does not redraw the game window, although the in-game cursor does appear. I then need to alt-tab back to Steam and close the game using the Stop button.

If the game is set to "Fullscreen Window", then alt-tabbing out of the game does not show me my desktop, I just get a non-interactive game screen until I alt-tab back to the game.

Windowed mode behaves as expected, although if I switch from Windowed mode, to Exclusive Fullscreen, the game window is not drawn correctly and I must shut it down.

Anyone seeing similar?

Info for reference:
System:
  Host: MintBox Kernel: 5.13.0-44-generic x86_64 bits: 64 
  Desktop: Cinnamon 5.2.7 Distro: Linux Mint 20.3 Una 
Graphics:
  Device-1: AMD driver: amdgpu v: kernel 
  Display: x11 server: X.Org 1.20.13 driver: amdgpu,ati 
  unloaded: fbdev,modesetting,radeon,vesa tty: N/A 
  OpenGL: renderer: AMD Radeon RX 6800 (sienna_cichlid LLVM 14.0.1 DRM 3.41 
  5.13.0-44-generic) 
  v: 4.6 Mesa 22.1.0 - kisak-mesa PPA

**Comment by hatani** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5845#issuecomment-1135511616

Nice! Exiting game really no longer hangs up the game.  I yet still have black screen issue described above, anybody else?

**Comment by abique** [comment]
Do you have an ETA for when it will reach proton experimental?

**Comment by hatani** [comment]
As I've written above, still having black screen with icons ingame, what else can I provide to help pinpoint the problem?

**Comment by purran23** [comment]
Hi,
I have also observed the same issue as @hatani with black screen if I use exclusive fullscreen and alt tab out of the game.

Running:
Arch
Kernel: 5.17.9-zen1-1-zen
AMD Ryzen 7 5800X
NVIDIA GeForce GTX 1080 Ti (driver version 515.43.04)

**Comment by Seibz** [comment]
Please use the bleeding edge branch of Proton Experimental

**Comment by hatani** [comment]
> Please use the bleeding edge branch of Proton Experimental

I do, and always did (for the scope of this case).  But the problem is still there:

![image](https://user-images.githubusercontent.com/740982/171804140-42e890da-adf7-4b1e-81fe-e3ccae10a444.png)
![image](https://user-images.githubusercontent.com/740982/171804330-86f2b3d2-0dad-4f49-a42f-58b5939c050f.png)

EDIT: problem was actually caused by forcing DX11!  I've seen that it helped some people play the game, and now after I deleted the command line argument, it works!

**Comment by villavic** [comment]
Is voice chat working for anyone? It's using Vivox apparently and I saw some fixes in other games ([comment in #5827](https://github.com/ValveSoftware/Proton/issues/5827#issuecomment-1126542829)) with similar issues & Vivox.
In my case (proton experimental, bleeding edge) I can hear people but my audio is never picked up, with or without push to talk enabled.

**Comment by villavic** [comment]
Voice chat now works!

**Comment by TheDromundKaas** [comment]
For about 8 days now (maybe slightly longer), V Rising is crashing pretty consistently, though not always in the same situation. Combat seems to trigger it, although I have crashed without it as well (though less frequently). The Unity Crash Handler is displayed shortly, afterwards the game terminates, and I can rejoin easily.

The last 2 reports on protondb.com suggest other people have similar problems.

I will try to add a proton log with a crash produced as soon as I have time to test more.

Edit 1: Created [a log](https://gist.github.com/TheDromundKaas/ba5662914da30553e90b8f45fb939f51), but it's 91MB, so I've uploaded only the start with the exception, which happened not even 2 minutes into the game. If more is necessary, I can provide more of the log (though creating a gist from it times out).

Edit 2: Unity Crash Handler didn't show up this time, I killed the process myself as it was not shutting down on its own.

Edit 3: Tried yesterday again with the settings described by the latest ProtonDB entry, this time forcing Proton 7.0.4 (didn't have 7.0.3 available) and using the command line _mesa_glthread=true gamemoderun %command%_, no crashes.

**Comment by praxis88** [comment]
For what it's worth this game doesnt work at all under any proton version for me

Here are the logs for GE, Expiramental, and 7

7 Says "idk how to directx11" and Expiramental and GE bring up the unity crasher app

[Glorious_Eggroll.log](https://github.com/ValveSoftware/Proton/files/10610194/Glorious_Eggroll.log)
[Expiramental.log](https://github.com/ValveSoftware/Proton/files/10610195/Expiramental.log)
[Proton7.log](https://github.com/ValveSoftware/Proton/files/10610196/Proton7.log)

https://gist.github.com/praxis88/58c201153d6714b2a5ccef9924e0e54d

**Comment by kisak-valve** [comment]
Hello @praxis88, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by praxis88** [comment]
@kisak-valve I appended it to the origional post

Reading through I realize libva_driver and vdpau driver were set to radeonsi ( because my real gfx card is on rma, and I've fallen back to an nvidia. I have never run v-rising for either card successfully). After removing the libva_driver and vdpau_driver declarations from /etc/environment, restarting the computer, and retesting, the behavior is the same.

**Comment by kisak-valve** [comment]
Thanks, we've seen AMDVLK break other Vulkan drivers before, and it's installed on your system causing the same to happen to you. ( https://gist.github.com/praxis88/58c201153d6714b2a5ccef9924e0e54d#file-gistfile1-txt-L300-L313 / https://gist.github.com/praxis88/58c201153d6714b2a5ccef9924e0e54d#file-gistfile1-txt-L565-L578 )

**Comment by praxis88** [comment]
@kisak-valve Thanks, that was the problem

**Comment by EcLiPzZz** [comment]
> 3. To fix intro video, launch game with:
>    `WINE_DO_NOT_CREATE_DXGI_DEVICE_MANAGER=1 %command%`


Sadly this didn't work for me, are there any other workarounds?

**Comment by intelligentgaming** [comment]
The game works with the latest build of Proton Experimental, however none of the videos will play, including the one in the background of the main menu.

However if you use GE-Proton instead, this fixes the videos.

**Comment by trsbox** [comment]
Got game working on linux with GE.  However, the game is too dark to play.  Anyone know of a launcher setting to fix this?  Using nvidia card

**Comment by rtngtom** [comment]
I had this game working up until a few days ago, so it doesn't have anything to do with the 1.0 update as far as I can tell, when it suddenly started to CTD any time a large number of effects occured, ie during most boss fights, it's possible a driver or library update caused something to go sideways, but I have no clue where to start. I've attached logs for Proton 9.0.1, Experimental, and GE-9.4, all of which experience the same crashes, as well as my system info according to Steam's System Information tool.

[Gist with Relevant Logs & Steaminfo](https://gist.github.com/rtngtom/980ec4d47f5f584d51231b8a3403a942)

**Comment by kisak-valve** [comment]
Hello @rtngtom, can you check if https://gitlab.freedesktop.org/drm/amd/-/issues/3343 is relevant to your system? A quick test would be to reboot to an older kernel and see how the game behaves.

**Comment by rtngtom** [comment]
@kisak-valve Yep, that's 100% it, downgrading to kernel 6.8.8-arch1-1 for current fixed the issue. Seems like I'll be waiting on that release, then. Thanks for the assist.

**Comment by tmzhuang** [comment]
I'm seeing an issue with this game and Planet Crafter where the game loads but I can't interact with the game (music plays but I can't click anything inside the game window).

I also run Valheim using proton experiment and that runs quite smoothly so not too sure what the difference is here with V Rising and Planet Crafter.

**Comment by kminehart** [comment]
* [System info](https://gist.github.com/kminehart/973696bf4fcb248aad3d467239fd0640)
* mesa 24.1.0-1
* Using KDE Plasma
* Proton version: Experimental (Forcing the game to use Proton 9.0 had the same results)

The issue I'm having that I haven't seen anyone report yet is most of the models in the game don't seem to render for me. The game runs super smooth without crashing but everything from walls to trees and rocks are just not rendered. :thinking: . Some things, like the tannery, chests, and grass (but not the furnace or workbenches) are rendered. Since I haven't seen anyone else have this issue I am assuming that I'm doing something wrong or weird here.

The only unique thing I can think of is I have all of my steam downloads in a separate drive (`/shared`, which is owned by `root:shared`) which is an XFS filesystem.

I have tried `systemctl start --user gamemoded` and using `gamemoderun %command%` but that had no effect.

Edit: Looks like this resolved for me after using the radeon vulkan implementation by installing the `vulkan-radeon` and `amd-vulkan-prefixes` packages and setting the game's command to `vk_radv %command%`

**Comment by wtblife** [comment]
Game runs fine for me in PopOS with proton 9, but the proton hotfix and proton 10 beta seem top keyboard input from being recognized by the game

**Comment by alasky17** [comment]
@wtblife could you give some more details?  Where exactly are you trying to use keyboard input and it is failing in the game?  Could you describe this more precisely?   

Also, could you please get a log by adding this as a launch option: PROTON_LOG=+hid,+hidp,+hid_report,+plugplay,+rawinput,+dinput,+xinput,+joystick,+setupapi,+input,+event,+key,+keyboard,+input %command%

And please also copy your system information from Steam (Steam -> Help -> System Information and Steam -> Help -> Steam Runtime Diagnostics) and put each in a [gist](https://gist.github.com/), then include a link to the gists in this issue report.

**Comment by tgerpheide** [comment]
The game wasn't recognizing any keyboard input aside from alt+f4 in both the main menu or anywhere in game. Trying to bind a new key in the input settings of the main menu was enough to test. I just experienced this same issue with one other game as well. I originally thought it was an issue with PopOS Cosmic Alpha, but it happened at the same time as when I updated and selecting Proton 9 over Proton Hotfix in steam seemed to immediately resolve the issue.

I'll try to get that info the next time I load the game

**Comment by alasky17** [comment]
@tgerpheide I just had another idea -- we just pushed a fix for another keyboard issue in a bump to Proton 10 Beta.  When you retest, please make sure you retest Proton 10.0-1d (which just went live ~30 seconds ago) just in case this is already fixed :)

**Comment by tgerpheide** [comment]
@alasky17 I did a quick test yesterday to no avail, but not confident it was using the updated version. Is 10.0-1d different from 10.0-1b in the compatibility settings? Should I be selecting experimental?

**Comment by wtblife** [comment]
@alasky17 No luck still, but not sure if experimental or 10.0-1 in Steam are the up to date versions (I tried both)

Deleting my other comments cause they were made on the wrong account

Here are the logs etc:
https://gist.github.com/wtblife/3ac4d04441348bcda2f896595df9d32b

**Comment by orlfman** [comment]
I'm having the weird issue that after 40 minutes or so the game's FPS tanks hard. Like 120fps down to 20fps for, what I can tell, no reason. I have to restart to "fix" it for another 40 minutes or so.

I'm running:
arch linux
linux64 6.14.6-arch1-1
mesa 1:25.1.0-1
vulkan-radeon 1:25.1.0-1

proton hotfix and proton experimental same issues. kde on wayland if that matters. intel 14900k and a 7900 xtx.

**Comment by Rastem88** [comment]
> I'm having the weird issue that after 40 minutes or so the game's FPS tanks hard. Like 120fps down to 20fps for, what I can tell, no reason. I have to restart to "fix" it for another 40 minutes or so.
> 
> I'm running: arch linux linux64 6.14.6-arch1-1 mesa 1:25.1.0-1 vulkan-radeon 1:25.1.0-1
> 
> proton hotfix and proton experimental same issues. kde on wayland if that matters. intel 14900k and a 7900 xtx.

The lag time bomb is due to https://github.com/ValveSoftware/steam-for-linux/issues/11446
The current workaround  is to enable the steam overlay or use LD_PRELOAD=""

**Comment by orlfman** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5845#issuecomment-2895016892

Thank you. Enabling overlay and screen recording did fix it. What an odd issue.

**Comment by kisak-valve** [comment]
V rising mouse movement issues on Pop_OS! 24.04

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9360.
@h-n-a posted on 2026-01-02T00:24:53:

# Compatibility Report
- Name of the game with compatibility issues: V rising
- Steam AppID of the game: 1604030

## System Information
- GPU: Radeon 860M
- Video driver version: 25.1.5-1
- Kernel version: 6.17.9
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/h-n-a/c7dc8770b899b5f097138861ae1a9a64

[steam-1604030.log](https://github.com/user-attachments/files/24403561/steam-1604030.log)

- Proton version: 10.0-3 (tested with other versions as well)

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
The camera won't rotate with the right mouse click as intended.

## Reproduction
-Install Pop_OS! 24.04
-Update the system "sudo apt update -y"
-Install steam .deb
-Install V Rising
-Use proton 10.0-3 (or any other version after 8)
-Host a game.
-Use the right mouse click and move the mouse around simultaneously, the camera doesn't rotate.

if i want to venture a guess, the mouse doesn't lock.
I tried using --force-grab-cursor flag, but that didn't make any difference.
